src/com/farzaneh/service/dao/ProductDAO.java

```diff
@@ -0,0 +1,180 @@
+package com.farzaneh.service.dao;
+
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
```

Comment:
for throwing exceptions use ProcessException instead of SQL Exception everywhere

Reply:
done

Reply:
done